docs/modules/install/pages/index.adoc

```diff
@@ -112,6 +112,34 @@ You can configure it with `crontab -e`, for instance if you've created your Deci
 4 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:reminders:all
 ----
```

Comment:
```suggestion
# Send notification mail digest daily
5 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_daily

# Send notification mail digest weekly on saturdays
5 0 * * 6 cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:mailers:notifications_digest_weekly
----
```

Mind that [this page](https://docs.decidim.org/en/install/) is a tutorial, so we shouldn't explain in detail any functionality. It is important that this is the shortest possible, so I suggest moving the two relevant lines to enable this and removing the rest. 

If we want to explain in detail the user facing feature we could do it in https://docs.decidim.org/en/admin/features/my_account/notifications_settings/ (mind that this'd be for users/admins, NOT for sysadmins, so nothing about cron or weird things like that)